sdks/python/apache_beam/coders/coder_impl.py

```diff
@@ -372,9 +337,11 @@ def encode(self, value):
 
 class FastPrimitivesCoderImpl(StreamCoderImpl):
   """For internal use only; no backwards-compatibility guarantees."""
-  def __init__(self, fallback_coder_impl):
+  def __init__(self, fallback_coder_impl, requires_deterministic=None):
     self.fallback_coder_impl = fallback_coder_impl
     self.iterable_coder_impl = IterableCoderImpl(self)
+    self.requires_deterministic = requires_deterministic
+    self.decoded_types = {}
```

Comment:
unused?

Reply:
Oops, this was leftover from before. This should fix the tests as well.